packages/react-aria-components/docs/Toolbar.mdx

```diff
@@ -0,0 +1,180 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:react-aria-components';
+import typesDocs from 'docs:@react-types/shared/src/events.d.ts';
+import {PropTable, HeaderInfo, TypeLink, PageDescription, ContextTable} from '@react-spectrum/docs';
+import styles from '@react-spectrum/docs/src/docs.css';
+import packageData from 'react-aria-components/package.json';
+import Anatomy from '@react-aria/toolbar/docs/toolbar-anatomy.svg';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+
+---
+category: Content
+keywords: [toolbar, aria]
+type: component
+---
+
+# Toolbar
+
+<PageDescription>{docs.exports.Toolbar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Toolbar']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/'}
+  ]} />
+
+## Example
+
+```tsx example
+import {Toolbar, Button, ToggleButton, Separator, Popover, MenuTrigger, Menu, MenuItem, Checkbox} from 'react-aria-components';
+
+<Toolbar>
+  <ToggleButton><strong>B</strong></ToggleButton>
+  <ToggleButton><div style={{textDecoration: 'underline'}}>U</div></ToggleButton>
+  <ToggleButton><i>I</i></ToggleButton>
+  <Separator orientation="vertical" />
+  <Button>Copy</Button>
+  <Button>Paste</Button>
+  <Button>Cut</Button>
+  <Separator orientation="vertical" />
+  <MenuTrigger>
```

Comment:
Something happened to the Menu popover:
<img width="366" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/dd020d3e-6d1a-480c-9c47-5cca860bd3e7">

Reply:
looks like the 30px padding applied on the Popover styles from the popover file is coming through now

Reply:
wha...??? it didn't look like that locally for me. looking into it

Reply:
o, right, merged the simplify docs css pr